SampleApp/SampleApp/ViewController.swift

```diff
@@ -92,15 +86,12 @@ class ViewController: UIViewController {
                 weakSelf.deviceAssessmentButton?.isEnabled = true
 
             }
-            
         }
-
-        
-        
     }
 
 }
-extension ViewController:DeviceRiskUploadCallback {
+
+extension ViewController: DeviceRiskUploadCallback {
     func dataUploadFinished(uploadResult: DeviceRiskUploadResult) {
         resultsTextView?.text = "UUID is \(uploadResult.uuid ?? "not generated")"
         deviceAssessmentButton?.isEnabled = true
```

Comment:
The code below here should also go away, since this code is asking the customer to store the ID

Reply:
done.